Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -25,7 +25,7 @@
 #import "ARKDefaultLogFormatter.h"
 #import "ARKLogStore.h"
 #import "ARKLogMessage.h"
-
+#import "Aardvark.h"
```

Comment:
This is happening because you're importing `ARKEmailBugReporter.h` before `ARKLogScreenshot` has been extern'd.